Source/ENS/Public/Characters/Player/EnsPlayerController.h

```diff
@@ -9,6 +9,9 @@
 #include "GameFramework/PlayerController.h"
 #include "EnsPlayerController.generated.h"
 
+class UCameraComponent;
+class UCapsuleComponent;
+class UStaticMeshComponent;
```

Comment:
just put class in front of the first usage. So, no need for top level fwd declarations.